InternxtDesktop/Services/ConfigLoader.swift

```diff
@@ -51,6 +56,7 @@ public struct ConfigLoader {
             let decodedData = try JSONDecoder().decode(JSONConfig.self, from: data)
             loadedConfig = decodedData
         } catch {
+            print(error)
```

Comment:
Is this `print` required?

Reply:
kinda, I'm replacing them by Sentry reports once I implement it, is just to dump the error stacktrace in the console for now